src/chrome/content/rules/Pingupod.de.xml

```diff
@@ -3,7 +3,6 @@
 	<!--	Direct rewrites:
 				-->
 	<target host="pingupod.de" />
-	<target host="www.pingupod.de" />
```

Comment:
This should be documented and redirected to non-www.
There is also a mismatch on mgp.pingupod.de and diaspora.pingupod.de can be addded as target to get redirected.